pkg/api/service/testing/make.go

```diff
@@ -20,6 +20,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/intstr"
 	utilpointer "k8s.io/utils/pointer"
+	"k8s.io/utils/ptr"
```

Comment:
you should get rid of all of the uses of `utilpointer` in the files you're touching

Reply:
I have replaced all instances of 'pointer' with 'ptr and The code has been re-committed.